server/routes/routers/tags.ts

```diff
@@ -20,6 +20,27 @@ tags.get('/', async (req: any, res: Response) => {
   }
 });
 
+//get all post tags
+tags.get('/all/post', async (req: Request, res: Response) => {
+  try {
+    const allPostTags = await prisma.tags.findMany(
+      {
+        where: {
+          tagType: 'Post'
+        }
+      }
+    )
+    res.send(allPostTags);
+  }
+  catch (err){
+    console.log('Error: GET /api/tags/all/post: ', err);
```

Comment:
Should be converted to console.error instead of console.log for consistent stderr logging